packages/unified-selection/src/unified-selection/SelectionScope.ts

```diff
@@ -113,7 +113,11 @@ async function* computeElementSelection(
     FROM AncestorElements
     WHERE ${recurseUntilRoot ? "" : "Depth = 0 OR"} ParentId IS NULL
   `;
-  yield* executeQuery(queryExecutor, { ctes, ecsql, bindings });
+  yield* executeQuery({
+    queryExecutor,
+    query: { ctes, ecsql, bindings },
+    config: { restartToken: `SelectionScope/compute-element-selection-query/${Guid.createValue()}` },
+  });
```

Comment:
- I think `computeSelection` should take an optional prop for the id (create one if not passed), and pass it down to these local functions.
- I'd create the `restartToken` in format: `computeSelection/{componentId}/element`